packages/react-components/react-button/src/stories/CompoundButton/CompoundButtonDefault.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { CompoundButton, CompoundButtonProps } from '@fluentui/react-button';
+import { CompoundButton, CompoundButtonProps } from '@fluentui/react-components';
```

Comment:
_nit:_ Separate props into `import type` statement

```suggestion
import { CompoundButton } from '@fluentui/react-components';
import type { CompoundButtonProps } from '@fluentui/react-components';
```